src/steamship/agents/memory/chathistory.py

```diff
@@ -97,7 +97,7 @@ def append_system_message(
             text=text, tags=tags, content=content, url=url, mime_type=mime_type
         )
 
-    def append_agent_message(
+    def append_assistant_message(
```

Comment:
We might want to start curating (in the docs) some common terminology, even if we know it'll change.

E.g. "we tend to use `assistant` to mean....", "we tend to use `xyz` to mean ..."